config/application.rb

```diff
@@ -23,7 +23,7 @@ module ServiceManualPublisher
 
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
```

Comment:
Once this version number has been bumped here, the entire `config/initializers/new_framework_defaults_7_0.rb` file can then be removed, as Rails will automatically load the configuration that you uncommented in the defaults file.